stable/artifactory-ha/templates/nginx-service.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "artifactory.nginx.name" . }}
+  labels:
+    app: {{ template "artifactory.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    component: {{ .Values.nginx.name }}
+spec:
+  type: {{ .Values.nginx.service.type }}
+{{- if eq .Values.nginx.service.type "LoadBalancer" }}
+  externalTrafficPolicy: Local
+{{- end }}
+{{- if .Values.nginx.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
+{{ toYaml .Values.nginx.service.loadBalancerSourceRanges | indent 4 }}
+{{- end }}
+  ports:
+  - port: {{ .Values.nginx.externalPortHttp }}
+    targetPort: {{ .Values.nginx.internalPortHttp }}
```

Comment:
It usually doesn't make much sense to make the contaienr port configurable. Changing it would certainly break things. Plus, I'd suggest you use a named port.

Reply:
Will remove. Should I also remove in templates/artifactory-service.yaml? Also there there is no good reason to change the defaults.

Reply:
Yes, do it everywhere. Plus, I'd suggest you use named ports and refer to them by their names in the services.